test/controllers/waitForOthersToSign.test.ts

```diff
@@ -0,0 +1,26 @@
+import 'reflect-metadata'
+
+import { assert } from 'chai'
+import { OK } from 'http-status-codes'
+import request from 'supertest'
+import { createApp } from './helpers/application.factory'
+import HtmlAssertHelper from './helpers/htmlAssert.helper'
+
+import 'app/controllers/waitForOthersToSign.controller'
+import { WAIT_FOR_OTHERS_TO_SIGN_URI } from 'app/paths'
+
+describe('WaitForOthersToSignController', () => {
+  describe('GET request', () => {
+    it('should render the WaitForOthers page', async () => {
+        const app = createApp()
```

Comment:
indent is inconsistent. Make sure to update your vscode preferences to 2-space indent